c2corg_ui/static/js/detailsview.js

```diff
@@ -0,0 +1,69 @@
+goog.provide('app.ViewDetailsController');
+goog.provide('app.viewDetailsDirective');
+
+goog.require('app');
+
+/**
+ * @return {angular.Directive} The directive specs.
+ * @ngInject
+ */
+app.viewDetailsDirective = function() {
+  return {
+    restrict: 'A',
+    controller: 'AppViewDetailsController',
+    controllerAs: 'detailsCtrl',
+    bindToController: true,
+    link: function(el, scope, attrs) {
+      // show tabs if they have been hidden on smartphone
+      $(window).resize(function() {
+        if (window.innerWidth > 620) {
```

Comment:
What about listing those width params in some JS file where they would be centralized?
I'm a bit concerned that if one day we want to change those values, we would have to search/replace in many files and possibly struggle because some setting would have been forgotten...